src/components/property-tree/property-tree.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from "react";
+// @ts-ignore
+import SortableTree from "react-sortable-tree";
+
+import { ValidChildAsset } from "../../utils/test-fixtures";
+
+import { Asset } from "@mtfh/common/lib/api/asset/v1/types";
+import "react-sortable-tree/style.css";
+
+interface PropertyTreeProps {
+  asset: Asset;
+  childAssets: Asset[] | ValidChildAsset[] | undefined;
+}
+
+interface TreeAsset {
+  title: React.ReactNode;
+  children: Array<TreeAsset> | null | { title: string; children: null };
+  expanded?: boolean;
+}
+
+export const PropertyTree = ({ asset, childAssets }: PropertyTreeProps): JSX.Element => {
+  const excludedTreeAssets = "656feda1-896f-b136-da84-163ee4f1be6c"; // Hackney Homes
+
+  const [treeViewData, setTreeViewData] = useState<TreeAsset[]>([]);
+
+  const addChildrenAssets = (): TreeAsset[] => {
+    const childrenAssets: TreeAsset[] = [];
+
+    if (childAssets) {
+      for (const [i, v] of childAssets.entries()) {
+        if (i < 5) {
+          childrenAssets.push(generateNode(v.assetAddress.addressLine1, [], v.id));
+        } else if (i === 5) {
+          childrenAssets.push({ title: "<< MORE.... >>", children: null });
+        } else {
```

Comment:
Arent both of these methods pushing nodes to a list? Could the generateNode method be used to include the bottom node type?

Reply:
I just discussed this with Dave. At the moment this makes is so that if there are over 4 children, we show a "MORE..." node, meaning the screen is not going to be filled with children assets (in case there's like 30+ children assets) which would take up vertical space and push other components in the page further down. We agreed that for the time being we'll be leaving this functionality as is.

Also, at the moment, we can't really see Block/Estate or any asset that's likely to have children in MMH, so I'm not sure how this is going to look. But we can then revisit this once MMH is enabled for all asset types.